cloudformation/cmp_uk.yaml

```diff
@@ -0,0 +1,117 @@
+            Principal:
+              Service: lambda.amazonaws.com
+            Action: sts:AssumeRole
+      Description: CloudWatch Synthetics lambda execution role for running canaries
+      ManagedPolicyArns: 
+        - arn:aws:iam::aws:policy/CloudWatchSyntheticsFullAccess
+      RoleName: !Sub CloudWatchSyntheticsRole-${CanaryName}-${AWS::Region}
+  RolePermissions:
+    Type: AWS::IAM::Policy
+    Properties:
+      Roles:
+        - Ref: CloudWatchSyntheticsRole
+      PolicyName: !Sub CloudWatchSyntheticsPolicy-${CanaryName}-${AWS::Region}
+      PolicyDocument:
+        Version: '2012-10-17'
+        Statement:
+          - Effect: Allow
+            Action:
+              - s3:PutObject
+              - s3:GetObject
+              - s3:GetBucketLocation
+            Resource:
+            - !Sub arn:aws:s3:::${S3Bucket}/*
+          - Effect: Allow
+            Action:
+              - logs:CreateLogStream
+              - logs:PutLogEvents
+              - logs:CreateLogGroup
+            Resource:
+              - !Sub arn:aws:logs:${AWS::Region}:${AWS::AccountId}:log-group:/aws/lambda/cwsyn-${CanaryName}-*
+          - Effect: Allow
+            Action:
+              - s3:ListAllMyBuckets
+              - xray:PutTraceSegments
+            Resource:
+              - "*"
+          - Effect: Allow
+            Action: cloudwatch:PutMetricData
+            Resource: "*"
+            Condition:
+              StringEquals:
+                cloudwatch:namespace: CloudWatchSynthetics
+  SyntheticsCanary:
+    Type: AWS::Synthetics::Canary
+    Properties:
+      ArtifactS3Location: !Sub s3://${S3Bucket}/canary/${AWS::Region}/${CanaryName}
+      Code:
+        Handler: pageLoadBlueprint.handler
+        S3Bucket: !Sub ${S3Bucket}
+        S3Key: !Sub ${Environment}/nodejs.zip
+      DeleteLambdaResourcesOnCanaryDeletion: true
+      ExecutionRoleArn:
+        Fn::GetAtt:
+          - CloudWatchSyntheticsRole
+          - Arn
+      Name: !Sub ${CanaryName}
+      RunConfig: 
+        TimeoutInSeconds: 120
+      RuntimeVersion: syn-nodejs-puppeteer-3.5
+      Schedule: 
+        Expression: rate(2 minutes)
+      StartCanaryAfterCreation: !If [IsProd, true, false]
+      Tags:
+        - Key: blueprint
+          Value: heartbeat
+  CloudWatchAlarm:
+    Type: AWS::CloudWatch::Alarm
+    Condition: IsProd
+    Properties:
+      ActionsEnabled: true
+      AlarmActions:
+          - !Sub arn:aws:sns:${AWS::Region}:${AWS::AccountId}:CommercialCanaries
+      AlarmDescription: Either a Front or an Article CMP has failed
+      AlarmName: UK CMP Failed
+      ComparisonOperator: LessThanOrEqualToThreshold
+      DatapointsToAlarm: 5
+      Dimensions:
+          - Name: CanaryName
+            Value: !Sub ${CanaryName}
+      EvaluationPeriods: 5
+      MetricName: SuccessPercent
+      Namespace: CloudWatchSynthetics
+      OKActions:
+          - !Sub arn:aws:sns:${AWS::Region}:${AWS::AccountId}:CommercialCanaries
```

Comment:
The `OkActions - !Sub` are:

`arn:aws:sns:${AWS::Region}:${AWS::AccountId}:AUSCommercialCanaries`
`arn:aws:sns:${AWS::Region}:${AWS::AccountId}:Commercial_CMP_CA_CloudWatch_Alarms_Topic`
`arn:aws:sns:${AWS::Region}:${AWS::AccountId}:CommercialCanaries`

If it's not a pain could we align the naming convention for these?

Reply:
Yeah absolutely. I'll add the topics to the template so they are recreated with a consistent naming convention.